streaming/src/message/message_bundle.h

```diff
@@ -74,18 +74,21 @@ class StreamingMessageBundleMeta {
                                                  bool verifer_check = true);
   inline virtual uint32_t ClassBytesSize() { return kMessageBundleMetaHeaderSize; }
 
+  inline static bool CheckBundleMagicNum(const uint8_t *bytes) {
+    const uint32_t *magic_num = reinterpret_cast<const uint32_t *>(bytes);
```

Comment:
Better to assert the bytes is not null?

Reply:
> Better to assert the bytes is not null?

`bytes` is already checked in FromBytes method.